### ISSUE TITLE
vcard:Kind instead of vcard:Individual

### ISSUE THREAD

**Issue by peterdesmet** [oSoc15/ipt-dcat#47]
In `adms:contactPoint`, the vcard is of type `Kind` instead of `Individual` (Individual is a good assumption for 99% of the contact points).

```
adms:contactPoint [ a vcard:Kind ; vcard:fn "Peter Desmet"  ; vcard:hasEmail <mailto:peter.desmet@inbo.be> ] ;
```

**Comment by simon-vc** [comment]
Kind is the super-class, if its true for 99% I'll change it to Individual

**Comment by peterdesmet** [comment]
You can set this to `Individual`

**Comment by peterdesmet** [comment]
Confirmed fixed in `v1.2`
